recipes/nipype/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "0.12.0" %}
+
+package:
+  name: nipype
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/nipy/nipype/archive/{{ version }}.tar.gz
+  md5: 4d010bae6028cf4622bc1946c679fabc
+
+build:
+  number: 0
+  # error: option --single-version-externally-managed not recognized
```

Comment:
So, if `setuptools` is an option, you can add it to the build requirements and add these back. Alternatively, leaving them out will go the `distutils` route. There is a slight preference of some people here for using`setuptools` if it is an option. I'm happy either way. Just trying to provide you info.